lib/hooks/api/useContributorList.ts

```diff
@@ -30,16 +30,15 @@ export const useContributorsList = ({
     meta: Meta;
   };
   initialPage?: number;
-  defaultLimit?: number;
+  defaultLimit?: string;
   defaultRange?: string;
 }) => {
-  const [range, setRange] = useState(defaultRange); // [start, end
```

Comment:
Refactor (nit): Prefer a separate pull request for the changes for range as it is not related to the current refactor.